static/app/views/replays/detail/console/index.tsx

```diff
@@ -87,10 +98,15 @@ function Console({breadcrumbs, startTimestampMs = 0}: Props) {
       ) : (
         <StyledEmptyMessage title={t('No results found.')} />
       )}
-    </Fragment>
+    </ConsoleContainer>
   );
 }
 
+const ConsoleContainer = styled('div')`
+  height: 100%;
+  overflow: auto;
+`;
```

Comment:
After moving the filters out of here, we might need to switch away from `height:100%` and use `<FluidHeight>` instead.

We can also
- throw a border on ConsoleTable
- check the padding/margin that's leftover on the bottom. everything should line up with the bottom of breadcrumbs when we're in sidebar layout. Check when scrolled to the top/bottom too

Reply:
> After moving the filters out of here, we might need to switch away from `height:100%` and use `<FluidHeight>` instead.
> 
> We can also
> 
> * throw a border on ConsoleTable
> * check the padding/margin that's leftover on the bottom. everything should line up with the bottom of breadcrumbs when we're in sidebar layout. Check when scrolled to the top/bottom too

Done! 

Result: 


https://user-images.githubusercontent.com/39612839/186025833-9918dc88-d15a-47ed-90b6-543111d1b84e.mp4